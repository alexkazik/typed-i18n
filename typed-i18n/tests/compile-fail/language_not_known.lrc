# hello
en Hello
se Hej
