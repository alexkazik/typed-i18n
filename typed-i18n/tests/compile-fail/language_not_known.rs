use typed_i18n_derive::TypedI18N;

#[derive(Copy, Clone, TypedI18N)]
#[typed_i18n(filename = "tests/compile-fail/language_not_known.lrc")]
#[typed_i18n(builder = "mixed_str")]
enum Language {
    En,
}

fn main() {}
