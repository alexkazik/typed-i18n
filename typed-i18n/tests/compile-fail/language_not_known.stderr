error: language se key hello is not known
 --> tests/compile-fail/language_not_known.rs:4:1
  |
4 | #[typed_i18n(filename = "tests/compile-fail/language_not_known.lrc")]
  | ^
